op-service/eth/types.go

```diff
@@ -142,8 +144,7 @@ type ExecutionPayload struct {
 	ExtraData     BytesMax32      `json:"extraData"`
 	BaseFeePerGas Uint256Quantity `json:"baseFeePerGas"`
 	BlockHash     common.Hash     `json:"blockHash"`
-	// nil if not present, pre-shanghai
-	Withdrawals *types.Withdrawals `json:"withdrawals,omitempty"`
+	Withdrawals   *Withdrawals    `json:"withdrawals,omitempty"`
```

Comment:
The withdrawals type shouldn't be included; it's a remnant of a previous Canyon iteration that the eip4844 branch is based on, develop is more up-to-date here, and uses the geth typing.